build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+buildscript {
+    repositories {
+        google()
+        maven(url = "https://plugins.gradle.org/m2/")
+    }
+
+    dependencies {
+        this.classpath("com.android.tools.build:gradle:4.1.2")
```

Comment:
this?

Reply:
That may just be a leftover from my auto-complete attempt. Auto-completion in Gradle script 😍

Reply:
Fixed ✅